src/sys/unix.rs

```diff
@@ -490,7 +534,7 @@ unsafe fn getsockopt<T>(fd: SysSocket, opt: c_int, val: c_int) -> io::Result<T>
 }
 
 /// Caller must ensure `T` is the correct type for `opt` and `val`.
-#[cfg(all(feature = "all", target_vendor = "apple"))]
+#[cfg(target_vendor = "apple")]
```

Comment:
Are you sure this won't cause warnings? I'm quite sure I added it for a reason.

Reply:
It's called in `_set_nosighup` now. CI would catch the warnings.